test/integration/plugins/test_quota.py

```diff
@@ -267,9 +257,8 @@ def test_07_quota(self):
         cmd.enddate = today 
         response = self.apiclient.quotaBalance(cmd)
 
-        self.debug("Quota Balance on: %s" % response.startdate)
```

Comment:
start date log needed?

Reply:
it is part of balance and not a tope level element, so "None" would be logged with the old message